tests/conftest.py

```diff
@@ -1602,12 +1602,24 @@ def fixture_apptag_rna() -> str:
     return "RNAPOAR025"
 
 
+@pytest.fixture(name="entry_id")
+def fixture_entry_id() -> int:
+    """Return an entry id attribute."""
+    return 1
```

Comment:
I would suggest using the value directly in tests instead of using a fixture for this